software/glasgow/applet/program/mec16xx/__init__.py

```diff
@@ -60,6 +60,8 @@ async def emergency_flash_erase(self):
         dr_reset_test.VTR_POR = 0
         await tap_iface.write_dr(dr_reset_test.to_bits())
 
+        self._logger.log(logging.WARNING, "After running emergency mass erase, a power cycle may be required on some chips.")
```

Comment:
I think we usually use something like

```suggestion
        self._logger.warn("After running emergency mass erase, a power cycle may be required on some chips.")
```

(I may be misremembering)